grails-app/views/stockMovement/_comments.gsp

```diff
@@ -1,6 +1,7 @@
 <section class="box" aria-label="Comments">
     <h2><warehouse:message code="comments.label"/></h2>
-    <g:if test="${requisition?.comments}">
+    <g:set var="comments" value="${stockMovement?.requisition?.comments ?: stockMovement?.shipment?.comments}" />
```

Comment:
so it is displaying requisition comments if any exist, otherwise shipment comments? But then below in the editComment and deleteComment actions always takes in stockMovementId. Can we not also edit requisition comments here?

Also, is there a need to display both requisition and shipment comments at the same time?

Reply:
at this moment we prioritize requisition over shipment, so if a stock movement has both, we would add a comment to the requisition.

Reply:
moreover - the requisition is always created before the shipment, so we shouldn't run into a situation where firstly we were adding comments to shipment (because requisition wouldn't exist), and then started adding comments to requisition.